docs/docs/cli/running-test-suites.md

```diff
@@ -1,38 +1,38 @@
-# Running Transactions From the Command Line Interface (CLI)
+# Running Test Suites From the Command Line Interface (CLI)
 
-Once you have created a transaction, whether from the Tracetest UI or via a text editor, you will need the capabity to run it via the Command Line Interface (CLI) to integrate it into your CI/CD process or your local development workflow.
+Once you have created a test suite, whether from the Tracetest UI or via a text editor, you will need the capabity to run it via the Command Line Interface (CLI) to integrate it into your CI/CD process or your local development workflow.
 
-The command to run a transaction is the same as running a test from the CLI.
+The command to run a test suite is the same as running a test from the CLI.
 
 The documentation for running a test via the CLI can be found here:
 
 - [tracetest run](./reference/tracetest_run.md): This page provides examples of using this command.
 
-## Running Your First Transaction
+## Running Your First Test Suite
 
-To run a transaction, give the path to the transaction definition file with the `'-f'` option. This will launch a transaction, providing us with a link to the created transaction run.
+To run a test suite, give the path to the test suite definition file with the `'-f'` option. This will launch a test suite, providing us with a link to the created test suite run.
```

Comment:
```suggestion
To run a Test Suite, give the path to the Test Suite definition file with the `'-f'` option. This will launch a Test Suite, providing us with a link to the created Test Suite run.
```